packages/ibm-products/src/components/Datagrid/useActionsColumn.js

```diff
@@ -81,8 +82,8 @@ const useActionsColumn = (hooks) => {
                           >
                             <OverflowMenu
                               {...rest}
+                              align={align || 'top'}
                               renderIcon={icon}
-                              hasIconOnly
```

Comment:
Removed. This was the source of the problem where we found a button was rendering inside another button. This wasn't normally visible because the duplicate was always perfectly placed above the origina.